public/js/components/SourceTabs.css

```diff
@@ -1,7 +1,8 @@
 .source-header {
   border-bottom: 1px solid var(--theme-splitter-color);
   height: 30px;
-  flex: 1;
+  flex: 0 0 30px;
```

Comment:
Correction on the basis. Set it to auto, otherwise we don't get the nice active tab is borderless on the bottom effect.